test/python/modules/linear_boltzmann_solvers/transport_keigen/keigenvalue_transport_2d_1b_qblock_cbc.py

```diff
@@ -57,6 +57,6 @@
         },
         sweep_type="CBC",
     )
-    k_solver = NonLinearKEigenSolver(lbs_problem=phys)
+    k_solver = NonLinearKEigenSolver(do_problem=phys)
```

Comment:
From the user's perspective, it feels strange to have to supply a value to a keyword that reads as "do (a) problem". I know we mean discrete ordinates, but at the user level, I am not certain this is how we should call it/expose it. @wdhawkins thoughts?

Reply:
I agree. We could just drop the prefix entirely and go with `problem=`.